core/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/common/decorator/SelectionSorterWeightFactory.java

```diff
@@ -18,14 +19,8 @@
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  * @param <T> the selection type
  */
+@Deprecated(forRemoval = true, since = "1.27.0")
```

Comment:
I believe that when something is deprecated, Javadoc requires a `@deprecated` tag as well.